### DISCUSSION TITLE
How to connect to local clients when vpn is connected

### DISCUSSION THREAD

**Discussion by dcihon** [pivpn/pivpn#1850] [Support]
I have Pivpn up and running . When connected outside of the local lan I can ping the local machine ip which is 192.168.68.44
I can also can ping the local vpn ip which is 10.10.238.1 .  What I can't do is ping any other computer on the local lan such as 192.168.68.15.
Here is my debug info.
`[~] demo@rhino $ pivpn -d
::: Generating Debug Output
[sudo] password for demo:
::::		PiVPN debug		 ::::
=============================================
::::		Latest commit		 ::::
Branch: master
Commit: 55ed0cb3a569237843c74ff888f453772f037e05
Author: Xavier L
Date: Tue Jun 4 11:45:37 2024 -0400
Summary: feat(install): add support for wireguard in containers
=============================================
::::	    Installation settings    	 ::::
PLAT=Ubuntu
OSCN=oracular
USING_UFW=0
pivpnforceipv6route=1
IPv4dev=wlan0
install_user=demo
install_home=/home/demo
VPN=wireguard
pivpnPORT=51820
pivpnDNS1=8.8.8.8
pivpnDNS2=8.8.4.4
pivpnHOST=REDACTED
INPUT_CHAIN_EDITED=0
FORWARD_CHAIN_EDITED=0
INPUT_CHAIN_EDITEDv6=
FORWARD_CHAIN_EDITEDv6=
pivpnPROTO=udp
pivpnMTU=1420
pivpnDEV=wg0
pivpnNET=10.10.238.0
subnetClass=24
pivpnenableipv6=0
ALLOWED_IPS="0.0.0.0/0, ::0/0"
UNATTUPG=0
INSTALLED_PACKAGES=(grepcidr net-tools bsdmainutils iptables-persistent wireguard-tools qrencode)
=============================================
::::  Server configuration shown below   ::::
[Interface]
PrivateKey = server_priv
Address = 10.10.238.1/24
MTU = 1420
ListenPort = 51820
### begin Manjaro ###
[Peer]
PublicKey = Manjaro_pub
PresharedKey = Manjaro_psk
AllowedIPs = 10.10.238.2/32
### begin iPhone ###
[Peer]
PublicKey = iPhone_pub
PresharedKey = iPhone_psk
AllowedIPs = 10.10.238.3/32
### end iPhone ###
=============================================
::::  Client configuration shown below   ::::
[Interface]
PrivateKey = Manjaro_priv
Address = 10.10.238.2/24
DNS = 8.8.8.8, 8.8.4.4

[Peer]
PublicKey = server_pub
PresharedKey = Manjaro_psk
Endpoint = REDACTED:51820
AllowedIPs = 0.0.0.0/0, ::0/0
=============================================
:::: 	Recursive list of files in	 ::::
::::	/etc/wireguard shown below	 ::::
/etc/wireguard:
configs
keys
wg0.conf

/etc/wireguard/configs:
Manjaro.conf
clients.txt
iPhone.conf

/etc/wireguard/keys:
Manjaro_priv
Manjaro_psk
Manjaro_pub
iPhone_priv
iPhone_psk
iPhone_pub
server_priv
server_pub
=============================================
::::		Self check		 ::::
:: [OK] IP forwarding is enabled
:: [OK] Iptables MASQUERADE rule set
:: [OK] WireGuard is running
:: [OK] WireGuard is enabled
(it will automatically start on reboot)
:: [OK] WireGuard is listening on port 51820/udp
=============================================
:::: Having trouble connecting? Take a look at the FAQ:
:::: https://docs.pivpn.io/faq
=============================================
:::: WARNING: This script should have automatically masked sensitive       ::::
:::: information, however, still make sure that PrivateKey, PublicKey      ::::
:::: and PresharedKey are masked before reporting an issue. An example key ::::
:::: that you should NOT see in this log looks like this:                  ::::
:::: YIAoJVsdIeyvXfGGDDadHh6AxsMRymZTnnzZoAb9cxRe                          ::::
=============================================
::::		Debug complete		 ::::
:::
::: Debug output completed above.
::: Copy saved to /tmp/debug.log
:::`

**Comment by orazioedoardo** [comment]
You are already supposed to connect to local clients. Perhaps you are connecting to the VPN from a 192.168.68.0/24 network too? Is 192.168.68.15 the only other PC you can try to ping? Know that windows doesn't respond to ping afaik.
